### PR TITLE
[CHORE 162642804] Implementing LFA feedback

### PR DESCRIPTION
**Pull request by lupamo3-zz**
**What does this PR do?**
Implements feedback given by the LFA.

**The feedback was as follows:**

1. Separating test data from test files.
2. Review of where I input the status codes.
3. Renaming variables to observe snake_case.
4. Replace make-response and jsonify on my responses.
5. Make the image and video fields file uploadable.
6. Made all the tests pass.

**Description of Tasks completed?**
Separated variable names with an underscore.
Separated test data from test files.
Renamed variables to observe snake_case.
Replaces make_response and jsonify on my responses. Apart from the fields where there was date data, I was unable to serialize the data without make_response and jsonify.
Removed status codes where they were two.

How should this be manually tested?
Clone the repository.
Follow the installation requirements on the readme
Test it on Postman.

**Blockers:**
 Tests not passing on Travis but passing locally.

_Alternatively:_
-Run the Heroku link to test working
[https://ireporterflask-api-heroku.herokuapp.com/](Heroku)

What are the relevant pivotal tracker stories?

#162642804

**Questions:**

### CODE REVIEW

File: app/api/v2/views/redflag_views.py [hunk 1]
```diff
@@ -66,23 +66,18 @@ def get(self, id):
                 }), 200)
 
         return {"error": ["Incident with that id not found"]}, 404
```
Comment (Bonifase) [inline app/api/v2/views/redflag_views.py:68]: why do you have this message in a list?

File: app/api/v2/views/redflag_views.py [hunk 2]
```diff
@@ -123,3 +118,22 @@ def patch(self, id):
...
+    def patch(self, id):
+        """ Admin makes changes to status"""
+        fetch_by_id = self.db.get_incident_by_id(id=id)
+        data = request.get_json(force=True)
+
+        if fetch_by_id:
+            self.db.update_status(data['status'], id)
+            return {"data": [{"New status": data['status'],
+                              "message": "Updated status successfully"}]}, 200
+        return {"data": [{"message": " Status Incident not found"}]}, 404
```
Comment (Bonifase) [inline app/api/v2/views/redflag_views.py:139]: return only one dictionary

File: app/api/v2/views/user_views.py [hunk 3]
```diff
@@ -35,26 +34,16 @@ class MyAdmin(Resource, UsersModel):
...
+    @jwt_required
+    def delete(self, id):
+        """ Docstring for deleting users"""
+        app_users = self.db.get_user_id(id)
+        if not app_users:
+            return {"error": "User with that id not found"}, 404
+        delete_user = self.db.delete_user(id)
+
+        if delete_user:
+            return {"data": {"id": id, "message": delete_user}}, 200
```
Comment (Bonifase) [inline app/api/v2/views/user_views.py:46]: return only one dictionary

File: app/tests/v2/test_base.py [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+            "comment": "",
+            "id": 1
+        }
+
+        self.token_login = {
+            "username": "Andela",
+            "password": "Eatlivecode"
+        }
+
+        self.auth = {
```
Comment (Bonifase) [inline app/tests/v2/test_base.py:58]: have your test data in a separate file

**Review by Bonifase — COMMENTED** [review]
You only test status codes, test response data and messages in your test files

File: app/api/v2/views/user_views.py (hunk 3 above)
Comment (lupamo3-zz) [reply app/api/v2/views/user_views.py:46]: Thank you for the feedback, I have resolved this.

File: app/tests/v2/test_base.py (hunk 4 above)
Comment (lupamo3-zz) [reply app/tests/v2/test_base.py:58]: I have separated the test data, by creating the test_base file. The only methods in the test_base file are for sign-up and login to generate token to authenticate the protected routes.

File: app/api/v2/views/redflag_views.py (hunk 2 above)
Comment (lupamo3-zz) [reply app/api/v2/views/redflag_views.py:139]: Thank you for the feedback, I have resolved this.

File: app/api/v2/views/redflag_views.py (hunk 1 above)
Comment (lupamo3-zz) [reply app/api/v2/views/redflag_views.py:68]: Thank you for the feedback, I have fixed this, I've removed lists from all responses.

**Pull request merged**